src/content/background.js

```diff
@@ -46,6 +44,10 @@ class TaskManager extends Object {
     return this.platformInfo.os;
   }
 
+  updateMaxBufferEntries() {
+    MAX_BUFFER_ENTRIES = 60 / this.interval_process_update * 5; // 5 minutes
```

Comment:
When we call this more often now lets remove this extra multiplication.

```suggestion
    MAX_BUFFER_ENTRIES = 300 / this.interval_process_update; // 5 minutes
```